example_annotate.py

```diff
@@ -49,7 +51,12 @@
         dataset = PMIDReader(args.pmids).read()
     elif os.path.exists(args.dir_or_file):
         print(warning)
-        dataset = TextFilesReader(args.dir_or_file).read()
+        #dataset = TextFilesReader(args.dir_or_file).read()
```

Comment:
Not good.

We need need ANOTHER subcase (another if else) to differentiate between text file vs. 2 files with .plain.html + .ann.json